### PR TITLE
feat: Navigation bar

### PR DESCRIPTION
**Pull request by tanishk15sharma**
Documentation (3)
**NAVIGATION / HEADER **
currently it have 3 nav links 

- HOME 
- ABOUT
- CONTACT

### CODE REVIEW

File: style/components/nav.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: rgb(80, 77, 77);
```
Comment (Jia-A) [inline style/components/nav.css:5]: maybe change the background color, it will look nicer and active

**Review by Jia-A — CHANGES_REQUESTED** [review]
Everything looks good...

File: style/components/nav.css (hunk 1 above)
Comment (tanishk15sharma) [reply style/components/nav.css:5]: sure I will look into it.

**Review by deepak29-git — COMMENTED** [review]
Everything looks good

File: style/components/nav.css (hunk 1 above)
Comment (amirnathani9) [inline style/components/nav.css:5]: give the variable name to the color instead of directly calling

**Comment by amirnathani9** [comment]
> All things look good. Try to give the live link in Discription.

**Pull request merged**